res/css/views/location/_LocationPicker.scss

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_LocationPicker {
+    width: 450px;
+    height: 500px;
+
+    border-radius: 4px;
+
+    display: flex;
+    flex-direction: column;
+}
+
+#mx_LocationPicker_map {
+    height: 324px;
```

Comment:
this seems very magical, whats the justification?

Reply:
"it looks pretty". will be replaced once we have an actual design.